packages/styles/src/skeleton.scss

```diff
@@ -0,0 +1,48 @@
+
+@import "./new-settings";
+@import "./mixins";
+
+$block: #{$fd-namespace}-skeleton;
+
+.#{$block} {
+  @include fd-reset();
+
+  display: inline-block;
+  cursor: wait;
+
+  &__canvas {
+    display: block;
+    fill: var(--sapContent_Placeholderloading_Background);
+  }
+
+  &__middle {
+    stop-color: var(--fdSkeletonGradientMiddleColor);
+    animation: fd-skeleton-shimmer 2s linear infinite;
+  }
+
+  &--animated {
+    .#{$block}__canvas {
```

Comment:
I don't get this specificity, `--fdSkeletonGradientMiddleColor` is part of `fd-skeleton__middle` element where you specify the stop color and the animation. Please check.

Reply:
ok, I got it :) It's because you have `<stop offset="0" stop-color="var(--sapContent_Placeholderloading_Background)"></stop>` which is a child element of `fd-skeleton__canvas`. Weird approach but I understand now. I would still use a class for the forst and last stop (like for the middle).